src/rp/util/number_spec.clj

```diff
@@ -0,0 +1,6 @@
+(ns rp.util.number-spec
+  (:require [rp.util.number :as util-number]
+            [clojure.spec :as s]))
```

Comment:
I took inspiration for this from https://github.com/rentpath/rp-query-clj/blob/master/src/rp/query/dsl_spec.clj - is this way this is used different?